src/cmd/sanity_test/sanity_tests.go

```diff
@@ -292,36 +295,88 @@ func checkAllSubFolder(
 // oneDrive
 // ---------------------------------------------------------------------------
 
-func checkOnedriveRestoration(
+func checkOneDriveRestoration(
 	ctx context.Context,
 	client *msgraphsdk.GraphServiceClient,
-	testUser,
-	folderName, dataFolder string,
+	userID, folderName, dataFolder string,
 	startTime time.Time,
 ) {
-	var (
-		// map itemID -> item size
-		fileSizes = make(map[string]int64)
-		// map itemID -> permission id -> []permission roles
-		folderPermission        = make(map[string][]permissionInfo)
-		restoreFile             = make(map[string]int64)
-		restoreFolderPermission = make(map[string][]permissionInfo)
-	)
+	drive, err := client.
+		UsersById(userID).
+		Drive().
+		Get(ctx, nil)
+	if err != nil {
+		fatal(ctx, "getting the drive:", err)
+	}
+
+	checkDriveRestoration(
+		ctx,
+		client,
+		userID,
+		folderName,
+		ptr.Val(drive.GetId()),
+		ptr.Val(drive.GetName()),
+		dataFolder,
+		startTime,
+		false)
+}
 
+// ---------------------------------------------------------------------------
+// sharePoint
+// ---------------------------------------------------------------------------
+
+func checkSharePointRestoration(
+	ctx context.Context,
+	client *msgraphsdk.GraphServiceClient,
+	siteID, folderName, dataFolder string,
+	startTime time.Time,
+) {
 	drive, err := client.
-		UsersById(testUser).
+		SitesById(siteID).
 		Drive().
 		Get(ctx, nil)
 	if err != nil {
 		fatal(ctx, "getting the drive:", err)
 	}
 
+	checkDriveRestoration(
+		ctx,
+		client,
+		siteID,
+		folderName,
+		ptr.Val(drive.GetId()),
+		ptr.Val(drive.GetName()),
+		dataFolder,
+		startTime,
+		true)
+}
+
+// ---------------------------------------------------------------------------
+// shared drive tests
+// ---------------------------------------------------------------------------
+
+func checkDriveRestoration(
+	ctx context.Context,
+	client *msgraphsdk.GraphServiceClient,
+	resourceOwner,
+	folderName,
+	driveID,
+	driveName,
+	dataFolder string,
```

Comment:
I've been waffling on that.  I think this format is okay, so long as the variables are all listed either in a single line, or each to their own line.  Thoughts?

Reply:
my nit here is that type information is only given once but params are split across lines so it's confusing to read. I tend to prefer having each param on a separate line with the type information on each line as well and the closing paren on a different line so the diff is smaller when parameters are added/removed. I know you have different preferences for some of those things

Overall, I'd prefer either

```go
folderName string,
driveID string,
driveName string,
dataFolder string,
```

or

```go
folderName, driveID, driveName, dataFolder string,
```

I personally find the former much easier to skim. The pattern currently in the PR makes skimming difficult because any given line does not contain all the information about the param name and the param type